### PR TITLE
Copy over the filepicker & fix some stuff

### LINKED ISSUE #7: File picker: when navigating to a directory list Y scroll is not reset
E.g. scroll to the bottom and open "sdcard", instead of displaying ".." at the top like it should, it's scrolled at the same (?) Y pos as the directory you navigated from.

### PR DESCRIPTION
**Pull request by xyzz**
Fixes #7 and #8

### CODE REVIEW

**Comment by xyzz** [comment]
apk: http://0x0.st/s8Ml.apk

File: app/src/main/java/is/xyz/mpv/MPVFilePickerFragment.java [hunk 1]
```diff
@@ -1,49 +1,33 @@
...
-        if (viewType != LogicHandler.VIEWTYPE_CHECKABLE)
-            return super.onCreateViewHolder(parent, viewType);
-        View v = LayoutInflater.from(getActivity()).inflate(R.layout.filepicker_item, parent, false);
-        return new CheckableViewHolder(v);
+    public void onClickCheckable(@NonNull View v, @NonNull FileViewHolder vh) {
+        mListener.onFilePicked(vh.file);
     }
 
     public boolean isBackTop() {
         return compareFiles(mCurrentPath, new File("/")) == 0;
```
Comment (sfan5) [inline app/src/main/java/is/xyz/mpv/MPVFilePickerFragment.java]: maybe restrict the top path to what was set in settings

**Comment by sfan5** [comment]
[MPL 2.0](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/LICENSE) is compatible with MIT I assume

lgtm

File: app/src/main/java/is/xyz/mpv/MPVFilePickerFragment.java (hunk 1 above)
Comment (xyzz) [reply app/src/main/java/is/xyz/mpv/MPVFilePickerFragment.java]: if we do then the user won't be able to navigate to e.g. external sd card (we have /sdcard as default now)

**Pull request merged**